src/main/java/nextstep/subway/path/domain/FareAgePolicy.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.subway.path.domain;
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+public enum FareAgePolicy {
+    TEENAGER(13, 19, 350, 0.8),
```

Comment:
```suggestion
    TEENAGER(age -> 13 <= age && age < 19, 350, 0.8),
```
 
취향차이지만..! 바로 조건을 넘겨줄 수도 있을 것 같습니다 ㅎㅎ